<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProceduresProcessingForm</class>
 <widget class="QMainWindow" name="ProceduresProcessingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение дела</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="nonIncludedProceduresTW">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="unincludeProcedureBut">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </widget>
   <widget class="QToolButton" name="includeProcedureBut">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>=&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="includedProceduresTW">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nonIncludedLB">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Включенные процедуры</string>
    </property>
   </widget>
   <widget class="QLabel" name="includedLB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Невключенные процедуры</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToCasesListBut">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
